Makefile

```diff
@@ -3,6 +3,12 @@ DM_ENVIRONMENT ?= local
 smoke-tests: setup
 	[ -f config/${DM_ENVIRONMENT}.sh ] && . config/${DM_ENVIRONMENT}.sh ; bundle exec cucumber --strict --tags @smoke-tests --tags ~@skip --tags ~@skip-${DM_ENVIRONMENT}
 
+smoulder-tests: setup
+	[ -f config/${DM_ENVIRONMENT}.sh ] && . config/${DM_ENVIRONMENT}.sh ; bundle exec cucumber --strict --tags @smoulder-tests,@smoke-tests --tags ~@skip --tags ~@skip-${DM_ENVIRONMENT}
+
+smoulder-tests-parallel: setup
+	[ -f config/${DM_ENVIRONMENT}.sh ] && . config/${DM_ENVIRONMENT}.sh ; bundle exec parallel_cucumber features/ -n 4 -o "--strict --tags @smoulder-tests,@smoke-tests --tags ~@skip --tags ~@skip-${DM_ENVIRONMENT} ${ARGS} -p run-parallel"
```

Comment:
So is this running `smoulder` and `smoke` in parallel, or running tests within each in parallel? Do we want to include smoke tests in these two commands?

Reply:
I thought yes, in the same way the functional tests also run the smoke tests - each being a superset of the next.